src/main/java/org/booking/controllers/FlightController.java

```diff
@@ -10,16 +10,22 @@
 import org.booking.utils.FileWorker;
 import org.booking.utils.Randomize;
 
+import java.time.LocalDate;
+import java.time.LocalTime;
 import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
 import java.util.List;
 
 public class FlightController implements IController {
     private final ServiceFlight service = new ServiceFlight();
 
     private long generateTime() {
-        return System.currentTimeMillis();
+        long currentTime = System.currentTimeMillis();
+        return currentTime / (15 * 60 * 1000) * (15 * 60 * 1000);
```

Comment:
return currentTime / (15 * 60 * 1000) * (15 * 60 * 1000);

What is the difference between currentTime / (15 * 60 * 1000) * (15 * 60 * 1000) values and currentTime value